wideq/client.py

```diff
@@ -7,9 +7,9 @@
 import requests
 import base64
 from collections import namedtuple
-from typing import Any, Optional
+from typing import Any, Dict, Generator, List, Optional
 
-from . import core
+from .core import Auth, Gateway, MonitorError, Session
```

Comment:
I think it would be awesome to keep these under the `core` namespace—I realize we use several things from that module, but I had been enjoying (for this particular case, not as a general rule) maintaining the core/client distinction by reminding myself where these come from when I refer to them.

Reply:
I ran into trouble with the tests, I'll try.